packages/react-effect/package.json

```diff
@@ -23,9 +23,6 @@
     "url": "https://github.com/Shopify/quilt/issues"
   },
   "homepage": "https://github.com/Shopify/quilt/blob/master/packages/react-effect/README.md",
-  "dependencies": {
-    "tslib": "^1.9.3"
```

Comment:
Where were we using this before? Was it just accidentally there?

Reply:
I'm a bit confused. The `tslib` [docs](https://github.com/Microsoft/tslib) mention 

> this means delivering users smaller files on average, as well as less runtime overhead.

Shouldn't having this be more beneficial? 🤔